solr/core/src/java/org/apache/solr/handler/component/SearchHandler.java

```diff
@@ -389,6 +389,9 @@ protected boolean checkCircuitBreakers(
     return false;
   }
 
+  public static final ThreadLocal<Boolean> DISABLE_REGION_CACHE =
```

Comment:
Not sure about this as well

Reply:
This is vestigial, it should be removed. Thanks for catching it!

Reply:
addressed with 8db36186b8ac96c74a44bc6627491f1346036b60